### DISCUSSION TITLE
Add support for DYAD - Capital efficient stablecoin

### DISCUSSION THREAD

**Discussion by Sovereignllama** [rotki/rotki#8459] [Ideas]
Add support for DYAD Notes and farming rewards.
![DYAD](https://github.com/user-attachments/assets/596b9259-7f82-4dd0-a7d6-9446d45a0085)

Twitter handle: https://x.com/0xDYAD

**Comment by yabirgb** [comment]
Got it, I'll add the asset to the next release and for the protocol we'll move this to an issue
